test/unit/create-organization-api-test.js

```diff
@@ -0,0 +1,58 @@
+import test from 'blue-tape'
+
```

Comment:
these look good. Just checking-- do other entities test sending query params (like limit?) if yes, let's add a test here. if not, let's skip it :)

Reply:
@phoebeschmidt I think they don't. As far as I can see.

Reply:
Although ... there are some intergration tests for entry that are testing against limit and skip.

Reply:
okay well then local testing is sufficient for me-- to make sure the SDK passes the params correctly to the API.